az-digital/az_quickstart
# Fixes #3030 Link Publication Titles to Node.

## Description
This PR adds functionality that causes generated citations to link the publication title to the node page if no other link applies. If the publication contains an uploaded file or external link, that will continue to be linked in preference to the node page.

## Related issues
#3030 

## How to test
- enable `az_publication`
- create publications
- create view of publications if desired
- verify that references link publication title to node page
- Give some publications a link or publication file
- Verify that the more specific links are chosen over the node link in those cases

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
